### DISCUSSION TITLE
Please make a guide for Windows?

### DISCUSSION THREAD

**Discussion by suparkoro86** [StevenBlack/hosts#2153] [Q&A]
Hi, I've used the hosts file on Linux to great success so first thank you to the authors of this github.  I used a youtube video to help install it on linux mint systems to make the computer safer for children to use.  

I've since given the children a newer Windows 10 PC and want to block all the same stuff.  Particularly, TIK TOK , DISCORD, and YOUTUBE SHORTS.  I am now looking to try and use hosts on Windows 10

Can someone write an easy to understand guide that anyone can understand?  Asking please as somebody who has no time or attention to figure it out himself.  If you can also include information on setting it to specifically only block certain social media but not other types, would be cool.  Thanks.

**Comment by StevenBlack** [comment]
Hello @suparkoro86 thank you for your question.

Have you read the section titled [Updating hosts file on Windows](https://github.com/StevenBlack/hosts) in the readme?  

We have a batch file named [`updateHostsWindows.bat`](https://github.com/StevenBlack/hosts/blob/master/updateHostsWindows.bat) that automates this.
